mongoose/templates/scenario-configmap.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-scenario-cm
+data:
+  scenario.js: |
+    {{ .Values.scenario }}
```

Comment:
Missing newline

Reply:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline